metricbeat/docs/reference/configuration/metricbeat-options.asciidoc

```diff
@@ -16,64 +13,62 @@ metricbeat.modules:
   hosts: ["root@tcp(127.0.0.1:3306)/"]
 ------------------------------------------------------------------------------
 
+//REVIEWERS: Note that I'm (mostly) removing comments from the code examples because we've found in the past that the commments get out of sync very quickly with the config files and become a maintenance problem.
 
 ==== Metricbeat Options
 
 You can specify the following options in the `metricbeat` section of the +{beatname_lc}.yml+ config file. These options
-are the same for all modules. Each module can have its specific additional configurations.
-For more details check the docs of each module.
+are the same for all modules. Each module may have additional configuration options that are specific to that module.
+See the <<metricbeat-modules>> section for more info about the available modules, the metricsets that they support,
+and any additional module-specific configuration info.
 
 ===== module
 
-The name of module that should be enabled.
+The name of the module to run. 
 
-===== metricset
-
-A list of metricsets that should be enabled for the module. Make sure to only list metricsets
-which are available in the module. It is not possible to reference metricsets from other modules.
-
-===== hosts
+===== metricsets
 
-A list of hosts to fetch information from. For some metricsets this is optional.
+A list of metricsets to execute. Make sure that you only list metricsets
+that are available in the module. It is not possible to reference metricsets from other modules.
 
 ===== enabled
 
-Boolean value if a the configuration block is enabled. If the configuration option is missing, it
-is enabled by default.
+A Boolean value that specifies whether the module is enabled. If you use the default config file, `metricbeat.yml`,
+the system module is enabled (set to `enabled: true`) by default. If the `enabled` option is missing from the
+configuration block, the module is enabled by default.
 
 ===== period
 
-Period on how often the metricsets should be called. This is required.
+How often the metricsets are executed. If a system is not reachable, Metricbeat returns an error for each period. This setting is required.
 
+===== hosts
+
+A list of hosts to fetch information from. For some metricsets, this setting is optional.
+
+===== fields
 
+A dictionary of fields that will be sent with the metricset event. This setting is optional. 
 
 ===== filters
 
-Each module configuration can have its own filters. More details on how to configure filters can be found here:
-<<configuration-filter>>.
+A list of filters to apply to the data generated by the module. For more detail on how to configure
+filters, see <<configuration-filter>>.
 
 ===== tags
 
-List of tags
-
-===== fields
+A list of tags that will be sent with the metricset event. This setting is optional. 
 
-Dict of fields
+//REVIEWERS: Is tags a valid option? I do not see it in the metricbeat.full.yml.
```

Comment:
Yes, it is a valid option. It exists for each module, but I would not add it to each module. Same for fields. Should we only add it to the system module as this is the default currently?